index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by the same entity.
```

Comment:
```suggestion
potentially making it complex to evaluate whether these roles are being filled
by appropriate entities.  Validation is the process by which verifiers apply business rules to 
evaluate the appropriateness of a particular use of a Verifiable Credential.
```

Subject==Holder is not the only acceptable validation rule. When my administrator gives my driver's license VC to my travel agent, the agent doesn't need to use cryptography to prove anything about the creator of the presentation. Rather, the verifier knows the business rules that apply, which find it acceptable to use a VC with a matching legal name, even if provided by someone other than the subject.